### DISCUSSION TITLE
C156167D-8519-6064-8E5C-55C604203513

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#150] [General]
# GN-z11

GN-z11 is an irregular galaxy located in the constellation of Ursa Major. At a distance of 32 billion light-years from Earth, it is the most distant and oldest galaxy ever discovered in the entire observable universe

https://celestia.mobi/addon?item=C156167D-8519-6064-8E5C-55C604203513

**Comment by T512-rgb** [comment]
Good add-on

**Reply by kmjordan122313-lgtm** [reply]
i agree.

**Comment by hacker-ARSHAVIR-XR4** [comment]
Did you guys not hear the news last week? A galaxy was found older than this one, it was discovered by JWST and is almost as old as the entire universe

**Reply by kmjordan122313-lgtm** [reply]
it is mom-z14, the most distant galaxy with a distance estimated at an approximate 34 billion light years.
